Library/Homebrew/version.rb

```diff
@@ -179,6 +179,19 @@ def self.detect(url, specs)
     end
   end
 
+  # TODO add tests if method is going to be left
+  def self.parse_version(val)
```

Comment:
I called it `parse_version`, because `parse` already defined in `Version`.
https://github.com/vladshablinsky/brew/blob/0ae9114dc1c621a51f7f73ff0fdb508fddf34d48/Library/Homebrew/version.rb#L283-L286